lib/src/over_react_redux/over_react_redux.dart

```diff
@@ -247,9 +317,41 @@ UiFactory<TProps> Function(UiFactory<TProps>) connect<TReduxState, TProps extend
       connectOptions.areMergedPropsEqual = allowInterop(handleAreMergedPropsEqual);
     }
 
+    dynamic interopMapStateToPropsHandler() {
```

Comment:
typed return as `dynamic` in case we ever want to allow for the `object` based syntax